src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -177,6 +193,167 @@
     @Column(name = "ADMISSION_REASON")
     private String admissionReason;
 
+    @OneToOne(mappedBy = "booking", cascade = CascadeType.ALL)
+    @PrimaryKeyJoinColumn
+    private ReleaseDetail releaseDetail;
+
+    public Optional<SentenceCalculation> getLatestCalculation() {
+        return sentenceCalculations.stream().max(Comparator.comparing(SentenceCalculation::getId));
+    }
+
+    public LocalDate getReleaseDate() {
```

Comment:
Feels like thee should be unit for this

Reply:
There are lots of integration tests but no unit test other than NonDtoReleaseDateTest